assets/app/features/upgrade/upgrade.constants.js

```diff
@@ -16,6 +16,7 @@
         .constant('PREPARE_TIMEOUT_INTERVAL', 1000)
         .constant('ADMIN_UPGRADE_ALLOWED_DOWNTIME', 30 * 60 * 1000)
         .constant('ADMIN_UPGRADE_TIMEOUT_INTERVAL', 5000)
+        .constant('SERVICES_TIMEOUT_INTERVAL', 1000)
```

Comment:
I have no idea how much time stopping services take. Most of those constants are meant to be tuned before release. Note that this interval is end-to-start time so it doesn't cause cascading. New check is scheduled only after successful response from the previous one.